arcgis-ios-sdk-samples/Route & Directions/Navigate route/NavigateRouteViewController.swift

```diff
@@ -93,9 +102,9 @@ class NavigateRouteViewController: UIViewController {
     
     /// Make the simulated data source for this demo.
     ///
-    /// - Parameter result: The solved route from the route task.
+    /// - Parameter route: The first route from `AGSRouteResult` from the route task.
```

Comment:
This description explains where the value came from, not how the value is used in this method. A description should help the user understand the purpose of the thing being described, so something like this might be better:

```suggestion
    /// - Parameter route: An `AGSRoute` object whose geometry is used to
    /// configure the data source.
```